Content.Shared/Damage/Components/DamageableComponent.cs

```diff
@@ -118,155 +119,133 @@ public override void HandleComponentState(ComponentState? curState, ComponentSta
                 return;
             }
 
-            _damageList.Clear();
+            _damageDict.Clear();
 
             foreach (var (type, damage) in state.DamageList)
             {
-                _damageList[type] = damage;
+                _damageDict[type] = damage;
             }
         }
 
+
+        // TODO QUESTION These 7 functions here are just wrapping standard function of PUBLIC dictionaries. I'm not sure
```

Comment:
This

Reply:
i think i'd prefer if the dictionaries were private, although they are already ReadOnly..